keras_rs/src/layers/embedding/distributed_embedding_test.py

```diff
@@ -685,6 +685,77 @@ def test_shared_table(self):
             res["feature3"].shape, (batch_size, EMBEDDING_OUTPUT_DIM)
         )
 
+    def test_mixed_placement(self):
+        if not self.on_tpu:
+            self.skipTest("Mixed placement test requires a TPU.")
+
+        # Use different embedding dimensions to verify that the correct tables
+        # are used for each feature.
+        embedding_output_dim1 = 16
+        embedding_output_dim2 = 32
+        embedding_output_dim3 = 64
+
+        # Intermix placement to exercise the change of order of inputs.
+        table1 = config.TableConfig(
+            name="table1",
+            vocabulary_size=VOCABULARY_SIZE,
+            embedding_dim=embedding_output_dim1,
+            placement="default_device",
+        )
+        table2 = config.TableConfig(
+            name="table2",
+            vocabulary_size=VOCABULARY_SIZE,
+            embedding_dim=embedding_output_dim2,
+            placement="sparsecore",
+        )
+        table3 = config.TableConfig(
+            name="table3",
+            vocabulary_size=VOCABULARY_SIZE,
+            embedding_dim=embedding_output_dim3,
+            placement="default_device",
+        )
+
+        embedding_config = {
+            "feature1": config.FeatureConfig(
+                name="feature1",
+                table=table1,
+                input_shape=(BATCH_SIZE_PER_CORE, 1),
```

Comment:
Why do we call it `BATCH_SIZE_PER_CORE`? Thought we changed the documentation to using `GLOBAL_BATCH_SIZE`?

Reply:
Oh, this is incorrect indeed. Will fix everywhere (not just this test) in a separate PR.